tests/integration_tests/cdc/cdc.go

```diff
@@ -20,7 +20,7 @@ import (
 	_ "github.com/go-sql-driver/mysql"
 	"github.com/pingcap/errors"
 	"github.com/pingcap/log"
-	"github.com/pingcap/tiflow/tests/integration_tests/dailytest"
+	"github.com/pingcap/tiflow/tests/integration_tests/cdc/dailytest"
```

Comment:
Why do we need to move it out? I guess it is only for tests.

Reply:
dailytest is only used in the cdc test, so it should not be placed in `tests/integration_tests`.